extra/1-currency-conversion.js

```diff
@@ -15,7 +19,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  let newPrice = ((price * 99) / 100) * 5.7;
+  let brazilPrice = newPrice.toFixed(2);
```

Comment:
Looks like the tests are failing here because this function is not returning a value. Any idea how you might fix this?